code/game/objects/items/ego_weapons/non_abnormality/jeong.dm

```diff
@@ -0,0 +1,59 @@
+//Jeong's Office - Grade 5, use in hand to cut your HP by 10%. Next attack deals 5x damage
+//The brightest stars last half as long
+/obj/item/ego_weapon/city/jeong
+	name = "jeong's office wakizashi"
+	desc = "A small blade used by Jeong's fixers"
+	special = "Use this weapon in hand to cut your HP by 10%. Next attack deals 5x damage."
+	icon_state = "jeong_fixer"
+	force = 30
+	attack_speed = 0.7
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("slices", "stabs")
+	attack_verb_simple = list("slice", "stab")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 60,
+							PRUDENCE_ATTRIBUTE = 60,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 60
+							)
+	var/ready = TRUE
+
+
+/obj/item/ego_weapon/city/jeong/attack_self(mob/living/carbon/human/user)
+	..()
+	if(!CanUseEgo(user))
+		return
+
+	if(!ready)
+		return
+	ready = FALSE
+	to_chat(user, "<span class='userdanger'>Low at Night.</span>")
+	force*=5
+	user.adjustBruteLoss(user.maxHealth*0.1)
+	addtimer(CALLBACK(src, .proc/Return, user), 5 SECONDS)
+
+/obj/item/ego_weapon/city/jeong/attack(mob/living/target, mob/living/carbon/human/user)
+	..()
+	if(force != initial(force))
+		to_chat(user, "<span class='userdanger'>High at Day.</span>")
+		force = initial(force)
```

Comment:
This is just infinitely better than the Blade Lineage one?

Reply:
Well, yeah, blade lineage was made way before. I can boost the damage cost of it to 20%